index.html

```diff
@@ -182,26 +183,56 @@ <h1>About me</h1>
+                            <li >
+                                <label for="fullname"></label>
+                                <input id="fullname" type="text" name="full_name" placeholder="full name" size="30" required>
+                            </li>
+                            <li>
+                                <label for="email"></label>
+                                <input id="email" type="email" name="email_name" placeholder="email address" size="25" required>
+                            </li>
+                            <li>
+                                <label for="textarea"></label>
+                                <textarea id="textarea" type="textarea" name="textarea" placeholder="Enter text here" size="500" required></textarea>
+                            </li>
                         </li>
-                    </li>
-                </ul>
-            <button class="button"> <a type="submit">Get in touch</a></button>
-            </form>
+                    </ul>
+                <button class="button"> <a type="submit">Get in touch</a></button>
+                </form>
+            </div>
+            <div class="contact-inner-label-desktop">
+                <p class="contact-par">I'm always interested in hearing about new projects, so if you'd like to chat get in touch</p>
+                <form action="https://formspree.io/f/xbjbownl" method="post">
+                    <ul class="first-item-inner-about-3">
+                        <ul class="form-about-1">
+                            <li class="form-about">
+                                <label for="firstname"></label>
+                                <input id="firstname" type="text" name="full_name" placeholder="first name" size="30" required>
+                            </li>
+                            <li class="form-about">
+                                <label for="lastname"></label>
+                                <input id="lastname" type="text" name="full_name" placeholder="last name" size="30" required>
+                            </li>
+                        </ul>
+                        <ul class="first-item-inner-about-4">
+                            <li class="form-about-li">
+                                <label for="email"></label>
+                                <input id="email" type="email" name="email_name" placeholder="email address" size="25" required>
+                            </li>
+                            <li class="form-about-li">
+                                <label for="textarea"></label>
+                                <textarea id="textarea" type="textarea" name="textarea" placeholder="Type something here..." size="500" required></textarea>
+                            </li>
+                        </ul>
+                    </ul>
+                <button class="button button-line" id="desk-button"> Get in touch</button>
+                </form>
+            </div>
```

Comment:
- [ ] As stated in the project requirement You must replicate the design as much as possible (e.g., font, colors, images, text, margins) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1), So I recommend you to adjust text font, background image, margin top and bottom for contact section just like the Figma design .

Your Project Design  contact section For Desktop screen size Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/193101950-dd0b3a62-95fa-4eb4-a5ef-e92075ea757f.PNG)

The Figma design Look likes this for Contact section : 
![Capture](https://user-images.githubusercontent.com/93598189/193102356-e4277564-a2f4-4853-82e4-4395e1756a83.PNG)